zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -66,13 +66,13 @@ import languageMap from '../assets/js/languageMap.json';
 import InputSelect from '../components/input_select/InputSelect';
 import Autocomplete from '../components/autocomplete/Autocomplete';
 import API from '../api';
-import { throttle } from '../utils.js';
+import { throttle } from '../utils.js/index.js';
 import Option from '../components/autocomplete/Option';
 import NotificationButton from '../components/notification_button/NotificationButton';
 import BreadCrumb from '../components/breadCrumb/breadCrumb';
 import DashboardLayout from '../layouts/DashboardLayout/DashboardLayout';
 import Navbar from '../components/Navbar/Navbar';
-import NotFoundPage from './not_found/NotFound';
+import NotFoundPage from '../views/not_found/NotFound';
```

Comment:
revert change. PageWrapper is already inside `views` folder. This was introduced by Berra but anyone that is available should please fix it